src/Microsoft.VisualStudio.Threading.Analyzers.Tests/MultiAnalyzerTests.cs

```diff
@@ -266,37 +284,43 @@ public void NoValueTupleReference()
         }
 
         [Fact]
-        public void NameOfUsedInAttributeArgument()
+        public async Task NameOfUsedInAttributeArgument()
         {
             var test = @"
 [System.Diagnostics.DebuggerDisplay(""hi"", Name = nameof(System.Console))]
 class Foo { }
 ";
-            this.VerifyCSharpDiagnostic(test);
+            await Verify.VerifyAnalyzerAsync(test);
         }
 
-        private void VerifyNoMoreThanOneDiagnosticPerLine(string test, bool hasEntrypoint = false)
+        public static class Verifier
         {
```

Comment:
This static class does not appear to be used. Why did you add it? Is the test framework you're adding using reflection to find it? (I hope not -- that's pretty hard to figure out).

Reply:
➡️ Verifiers are aliased to `Verify`. In this case it's on line 14.

Reply:
OK, I see that now. Why not just name it `Verify` and skip the redirection?

Reply:
This is a point of ongoing discussion. 😁